src/System.Net.Sockets/src/System/Net/Sockets/_ReceiveMessageOverlappedAsyncResult.Windows.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Nit: Same general comment: this isn't really separating a PAL layer, instead it's just splitting a lot of files in .Windows and .Linux versions.